### PR TITLE
idea-ultimate: v2023.1

### PR DESCRIPTION
**Pull request by KannarFr**

### CODE REVIEW

File: packages/dev-util/idea-ultimate/idea-ultimate-2023.1.exheres-0 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+DOWNLOADS="https://download.jetbrains.com/idea/ideaIU-${PV}.tar.gz"
+
+LICENCES="Apache-2.0"
+SLOT="0"
+PLATFORMS="~amd64"
+MYOPTIONS=""
+
+src_unpack() {
+    default
+    edo mv idea-IU-* ${PNV}
```
Comment (sardemff7) [inline packages/dev-util/idea-ultimate/idea-ultimate-2023.1.exheres-0:17]: Why not set `WORK`?

File: packages/dev-util/idea-ultimate/files/idea-ultimate.desktop [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Based on autogenerated by IDEA 2020 desktop entry
```
Comment (sardemff7) [inline packages/dev-util/idea-ultimate/files/idea-ultimate.desktop:1]: So it’s generated somewhere?

Comment (KannarFr) [reply packages/dev-util/idea-ultimate/files/idea-ultimate.desktop:1]: image
IntelliJ itself has a tool to create it. I just copied it and adapted it (a bit).

File: packages/dev-util/idea-ultimate/idea-ultimate-2023.1.exheres-0 (hunk 1 above)
Comment (KannarFr) [reply packages/dev-util/idea-ultimate/idea-ultimate-2023.1.exheres-0:17]: I should have asked for your review sorry.

I just copied it from idea-community package. Can you make a code suggestion? :D

**Pull request merged**